examples/zephyr/certificate_provisioning/boards/nrf52840dk_nrf52840.conf

```diff
@@ -13,3 +13,6 @@ CONFIG_MCUMGR_GRP_FS_FILE_ACCESS_HOOK=y
 
 # Allow reboot from shell after loading certificates
 CONFIG_REBOOT=y
+
+CONFIG_NET_BUF_DATA_SIZE=256
```

Comment:
For the fw_update sample we set `CONFIG_NET_BUF_TX_COUNT=20` instead of changing `CONFIG_NET_BUF_DATA_SIZE`. I don't really have an opinion as to which is the better one to change, but would like to keep the examples consistent if possible. Have/can you try with `CONFIG_NET_BUF_TX_COUNT=20`?

Reply:
This symbol is already resolves to 32 by for this application:
```
CONFIG_NET_BUF_TX_COUNT=32
```

I found that the change to NET_BUF_DATA_SIZE was necessary to get the handshake to complete successfully.

Reply:
Cool, let's do it.